source/test/sgp_mode_test/ParentTask.test.cc

```diff
@@ -0,0 +1,71 @@
+#include "../../sgp_mode/GenomeLibrary.h"
```

Comment:
Looks like similar recurring issues in this file:
* tag names
* spelling of symbionts
* remove all prints
* think about restructuring to target more specific functionality